redbot/cogs/trivia/data/lists/technology.yaml

```
@@ -0,0 +1,205 @@
+- Cohen
+- Fred Cohen
+- Fred
+A computer virus that actively attacks an anti-virus program or programs in an effort to prevent detection.:
+- Retrovirus
+In what year did the Symantec Corporation first release Norton Antivirus?:
+- 1990
+In 1988 one of the most common viruses was unleashed. Activated every Friday the 13th, the virus affects both .EXE and .COM files and deletes any programs run on that day. What is the name of that virus?:
+- Jerusalem
+Before a Cascading Style Sheet is useful, you need to include one into your HTML page. What HTML tag can you use to apply CSS rules to a document?:
+- Style
+Which SQL command would you use to retrieve a record or records from the database?:
+- SELECT
+- Select
+Which SQL command would you use to add a new record (row) to the database?:
+- INSERT
+- Insert
+Which SQL command would you use to modify the contents of an existing record (row) in the database?:
+- UPDATE
+- Update
+Since there are only 10 different digits, we can't have any base greater than 10.:
+- False
+- No
+What would be the decimal value of 1111 1110?:
+- 254
+A double is classified as what data type?:
+- A floating point data type
+- Floating point data type
+What is the symbol for the modulus operator?:
+- "%"
+What function is automatically called at the beginning of a C++ program?:
+- main()
+What is the standard file extension for a C++ source code file?:
+- .cpp
+Who was the child of a famous poet and English mathematician whom many historians consider the first programmer?:
+- Ada Lovelace
+- Ada
+- Lovelace
+What is the name of the first widely-used programming language that was developed by IBM in the mid 1950s?:
+- Fortran
+What is the name of the programming language, created in 1958, popularly used in artificial intelligence and which uses lists as a main data structure?:
+- LISP
+What is a function declaration without a body called?:
+- Function prototype
+What is a parameter that receives the location of an argument called?:
+- Reference parameter
+Which 32-bit register is used as the default loop counter?:
+- ECX
+Which operator in assembly language counts the number of bytes in an identifier?:
+- TYPE
+What does the "Q" in QBasic stand for?:
+- Quick
+QBasic was developed by which company?:
+- Microsoft
+The word COBOL is an acronym for what?:
+- COmmon Business Oriented Language
+- Common business oriented language
+Why was the language called "Python "?:
+- The inventor is a fan of "Monty Python's Flying Circus"
+- Monty Python's Flying Circus
+Which languages does Python owe its origin to?:
+- ABC, Modula-3
+- ABC
+- Modula-3
+- Modula 3
+What is the Java implementation of Python popularly known as ?:
+- Jython
+If you wanted to capitalize the first letter of each word in a string in PHP, which function would you use?:
+- ucwords()
+How would I decorate a class method if I wanted to use it without initializing a class?:
+- @staticmethod
```

```suggestion
- "@staticmethod"
```
Similar error as before, this needs quotes since it starts with an `@`.